src/span.js

```diff
@@ -242,6 +241,8 @@ export default class Span {
         let tags = [];
         for (let i = 0; i < this._tags.length; i++) {
             let tag = this._tags[i];
+            let emptyBuffer = new Buffer(1);
+            emptyBuffer.fill(0);
             tags.push({
                 key: tag.key,
                 tagType: 'STRING', // TODO(oibe) support multiple tag types
```

Comment:
next line - you need to convert the value to string if it's not a string already